Packs/CrowdStrikeFalconStreamingV2/Integrations/CrowdStrikeFalconStreamingV2/CrowdStrikeFalconStreamingV2.py

```diff
@@ -167,7 +171,13 @@ async def fetch_event(self, initial_offset: int = 0, event_type: str = '') -> As
                             if stripped_line:
                                 events_fetched += 1
                                 try:
-                                    yield json.loads(stripped_line)
+                                    streaming_event = json.loads(stripped_line)
+                                    event_metadata = streaming_event.get('metadata', {})
+                                    event_creation_time = event_metadata.get('eventCreationTime', 0) / 1000
+                                    event_creation_time_dt = datetime.fromtimestamp(event_creation_time)
+                                    if event_creation_time_dt < first_fetch_time:
+                                        continue
```

Comment:
Also here add a debug log that event X with timestamp Y was skipped.

Reply:
wanted to add it but was concerned with the fact it might add thousands of log lines
think its ok?

Reply:
I think it is ok. As long as it is in debug